seatunnel-connectors-v2/connector-cdc/connector-cdc-sqlserver/src/main/java/org/apache/seatunnel/connectors/seatunnel/cdc/sqlserver/source/utils/SqlServerTypeUtils.java

```diff
@@ -32,7 +32,15 @@
 
 /** Utilities for converting from SqlServer types to SeaTunnel types. */
 public class SqlServerTypeUtils {
+    private static final String DATETIME_OFFSET = "datetimeoffset";
+
     public static SeaTunnelDataType<?> convertFromColumn(Column column) {
+
+        switch (column.typeExpression()) {
+            case DATETIME_OFFSET:
+                return LocalTimeType.LOCAL_DATE_TIME_TYPE;
+        }
```

Comment:
I don't know if this code style is a better choice. Please correct me if you have any optimization suggestions
PTAL: @ic4y @hailin0 @EricJoy2048

Reply:
> I don't know if this code style is a better choice. Please correct me if you have any optimization suggestions PTAL: @ic4y @hailin0 @EricJoy2048

I don't think this is a good code. 
1. Only one option is not suitable for using switches
2. There is no default option in the switch, which will result in unpredictable behavior.

Reply:
Do you have any good suggestions about this?  Can you provide some ideas
@EricJoy2048

Reply:
Agree with Eric. I don't understand why the "switch-cases" construct is used when there is only one case. Why not using "if - return"?